lib/node_modules/@stdlib/math/base/special/README.md

```diff
@@ -323,7 +323,7 @@ var fcns = special;
 -   <span class="signature">[`rsqrtf( x )`][@stdlib/math/base/special/rsqrtf]</span><span class="delimiter">: </span><span class="description">compute the reciprocal of the principal square root of a single-precision floating-point number.</span>
 -   <span class="signature">[`secd( x )`][@stdlib/math/base/special/secd]</span><span class="delimiter">: </span><span class="description">compute the secant of an angle measured in degrees.</span>
 -   <span class="signature">[`sici( x )`][@stdlib/math/base/special/sici]</span><span class="delimiter">: </span><span class="description">compute the sine and cosine integrals.</span>
--   <span class="signature">[`spence( x )`][@stdlib/math/base/special/spence]</span><span class="delimiter">: </span><span class="description">spence’s function, also known as the dilogarithm.</span>
+-   <span class="signature">[`spence( x )`][@stdlib/math/base/special/spence]</span><span class="delimiter">: </span><span class="description">spence's function, also known as the dilogarithm.</span>
```

Comment:
@Planeshifter Looks like Spence gets mistakingly uncapitalized. I'll go ahead and approve, but ideally this should get fixed in whatever lint rule needs to be updated with an exception.